crates/holochain/src/conductor/tests/cell_cloning.rs

```diff
@@ -139,6 +140,58 @@ async fn create_clone_cell_run_twice_returns_correct_clones() {
     assert_eq!(clone_cell_1.original_dna_hash, dna.dna_hash().to_owned());
 }
 
+#[tokio::test(flavor = "multi_thread")]
+async fn create_identical_clone_cell_twice_fails() {
+    let (dna, _, _) = SweetDnaFile::unique_from_test_wasms(vec![TestWasm::Create]).await;
+    let role_name: RoleName = "dna_1".to_string();
+    let mut conductor = SweetConductor::from_standard_config().await;
+    let alice = SweetAgents::one(conductor.keystore()).await;
+    let app = conductor
+        .setup_app_for_agent("app", alice.clone(), [&(role_name.clone(), dna.clone())])
+        .await
+        .unwrap();
+    let clone_cell_payload = CreateCloneCellPayload {
+        app_id: app.installed_app_id().clone(),
+        role_name: role_name.clone(),
+        modifiers: DnaModifiersOpt::none().with_network_seed("seed".to_string()),
+        membrane_proof: None,
+        name: None,
+    };
+
+    let clone_cell = conductor
+        .clone()
+        .create_clone_cell(clone_cell_payload.clone())
+        .await
+        .unwrap();
+
+    let identical_clone_cell_err = conductor
+        .clone()
+        .create_clone_cell(clone_cell_payload.clone())
+        .await;
+    matches!(
+        identical_clone_cell_err,
+        Err(ConductorError::AppError(AppError::DuplicateDnaHash(dna_hash))) if dna_hash == *clone_cell.cell_id.dna_hash()
+    );
+
+    // disable clone cell and try again to create an identical clone
+    conductor
+        .clone()
+        .disable_clone_cell(&DisableCloneCellPayload {
+            app_id: app.installed_app_id().clone(),
+            clone_cell_id: CloneCellId::CellId(clone_cell.cell_id.clone()),
+        })
+        .await
+        .unwrap();
+    let identical_clone_cell_err = conductor
+        .clone()
+        .create_clone_cell(clone_cell_payload)
+        .await;
+    matches!(
+        identical_clone_cell_err,
+        Err(ConductorError::AppError(AppError::DuplicateDnaHash(dna_hash))) if dna_hash == *clone_cell.cell_id.dna_hash()
+    );
+}
+
```

Comment:
To be safe, there should be a test that creating the same clone on multiple apps (different agents) succeeds.